doc/src/pages/edit.tsx

```diff
@@ -20,18 +22,26 @@ const Edit: FC = () => {
     const path = getPath();
 
     setIsLoading(true);
+
+    if (path === '' || !path.includes('https://github.com/apache')) {
+      setPathExist(false);
+      setIsLoading(false);
+      return;
+    }
```

Comment:
You can move this part of the code up 24 lines, you should not need to `setIsLoading(false)` this way.

Also `if(xxx)` can be written as `if(!path?.includes(INDEX_URL))` 🤔

Reply:
Thanks for the review, @SkyeYoung. I'll move that code up and use `if (!path?.includes(INDEX_URL))` as you suggested. By the way, I'm currently using `INDEX_URL` as `https://github.com/apache/apisix` is that okay? Thanks!

Reply:
I think it's okay.